Various fixes in the guides

cescoffier opened the pull request:
Fix #1356 - fix wording in the native guide
Fix quarkusio/quarkusio.github.io#37 - Remove my name from the directories
Fix quarkusio/quarkusio.github.io#116 - Change location of the docker file

cescoffier committed:
Fix https://github.com/quarkusio/quarkus/issues/1356

Update the title of the guide as well a the wording in the content
Note: The filename was not updated on purpose (because of links)

cescoffier committed:
Fix https://github.com/quarkusio/quarkusio.github.io/issues/37

Replace my name with $HOME :-(

cescoffier committed:
Fix https://github.com/quarkusio/quarkusio.github.io/issues/116

Update the OpenShift instructions to copy the docker file to the root directory
Also update the Kubernetes instructions as the file was moved.

gsmet commented on docs/src/main/asciidoc/building-native-image-guide.adoc:
```diff
@@ -1,5 +1,5 @@
 include::./attributes.adoc[]
-= {project-name} - Building a Native Image
+= {project-name} - Building a Native Executable
```
Should we change the guide file name too?

I'm pretty sure we are also using native image in other guides, don't we? Should we change it everywhere? Or did you already check?

cescoffier replied:
It's been reported on this one, but better check, I will let you know.

Changing the file name means changing the links, as the links have been sent around a few times already, better not changing it already. I plan to change it during the release process.

gsmet commented on docs/src/main/asciidoc/building-native-image-guide.adoc:
```diff
@@ -85,7 +85,7 @@ If you have generated the application from the previous tutorial, you can find i
 ----
 
-We use a profile because, you will see very soon, packaging the native image takes a _few_ seconds.
+We use a profile because, you will see very soon, packaging the native executable takes a _few_ seconds.
```
You didn't fix the issue with the filename of the runner?

We really need to have consistency between:
- I run using the guide instructions
- I get the solution from the quickstarts repo

cescoffier replied:
Forgot that one! Will fix it in the same PR.

cescoffier replied:
So, I thought about it and came up with a third solution - delete everything ;-)

With the current content of the generated projects; no need to have 3 sub-projects in quickstarts, only a single one is required. It avoids the issue and makes things much more fluid. 

This is done in: https://github.com/quarkusio/quarkus-quickstarts/pull/87

gsmet reviewed (approved):
Looks good so feel free to merge. I added 2 questions.

cescoffier committed:
Replace native images with native executable in the guides

cescoffier committed:
Adapt pictures use in guide to be closer to the web site theme

cescoffier committed:
Fix callout

cescoffier committed:
Remove banner from screenshot as it does not fit with the web site theme

cescoffier committed:
Remove banner from screenshot as it does not fit with the web site theme as well as a reference to shamrock

cescoffier committed:
Add a script doing a selective copy of the guide to update the web site.

cescoffier committed:
Add a note in the getting-started guide so the user don't forget to stop the application

cescoffier committed:
Fix the native guide to be consistent with the generated project and quickstart projects

Also remove Scott name from the path

cescoffier committed:
Use oc patch to avoid having to copy the docker file to the project root

The proposal comes from https://github.com/quarkusio/quarkusio.github.io/pull/133.

cescoffier committed:
Merge branch 'master' of github.com:quarkusio/quarkus into features/fix-openshift-instructions-dockerfile-location

cescoffier committed:
Fix admonition in the fault-tolerance guide

cescoffier commented:
I have also applied the fix proposed in https://github.com/quarkusio/quarkusio.github.io/pull/133 by @sabre1041 

Before I forgot, this PR also contains a bash script to do the selective copy required to keep the web site in sync.

cescoffier commented:
@gsmet let me know if it's ok for you.

cescoffier committed:
Forgot to actually add the files to staging when updating the web site

cescoffier committed:
Fix for the typocs reported in https://github.com/quarkusio/quarkusio.github.io/pull/127

gsmet commented on docs/src/main/asciidoc/extension-authors-guide.adoc:
```diff
@@ -517,29 +517,29 @@ public class PersistenceAndQuarkusConfigTest {
 
 `io.quarkus.deployment.builditem.substrate.SubstrateSystemPropertyBuildItem`::
-A system property that will be set at native image build time.
+A system property that will be set at executable image build time.
```
Why do you use `executable image` here and below?

I could see how we could use `native image` here instead of `native executable` but `executable image` looks weird.

gsmet reviewed (commented):
@cescoffier I added a question inline. The rest looks good to me.

cescoffier committed:
Fix the usage of native image

Replaced with native executable.
